new-log-viewer/src/utils/file.ts

```diff
@@ -22,4 +22,21 @@ const openFile = (onOpen: OnFileOpenCallback) => {
     input.click();
 };
 
-export {openFile};
+/**
+ * Triggers a download of the provided Blob object with the specified file name.
+ *
+ * @param blob The Blob object to download.
+ * @param fileName The name of the file to be downloaded.
+ */
+const downloadBlob = (blob: Blob, fileName: string) => {
+    const url = URL.createObjectURL(blob);
+    const link = document.createElement("a");
+    link.href = url;
+    link.download = fileName;
+    link.click();
+    URL.revokeObjectURL(url);
+};
+
```

Comment:
Let's move this above `openFile` so the function declarations follow the order of the exports.